lib/config.ts

```diff
@@ -356,8 +356,18 @@ export interface Config {
   baseUrl?: string;
 
   /**
-   * CSS Selector for the element housing the angular app - this defaults to
-   * 'body', but is necessary if ng-app is on a descendant of <body>.
+   * A CSS selector for the DOM element you want Protractor to hook into your
+   * app at.
```

Comment:
"Protractor will use bootstrapping" is a little unclear but that's definitely an improvement